### PR TITLE
server init

### PR DESCRIPTION
**Pull request by andr-dev**

### CODE REVIEW

File: server/src/cors.rs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+            name: "Add CORS headers to responses",
+            kind: Kind::Response,
+        }
+    }
+
+    async fn on_response<'r>(&self, _request: &'r Request<'_>, response: &mut Response<'r>) {
+        response.set_header(Header::new("Access-Control-Allow-Origin", "*"));
+        response.set_header(Header::new(
+            "Access-Control-Allow-Methods",
+            "POST, GET, PATCH, OPTIONS",
```
Comment (leeminkyu16) [inline server/src/cors.rs:20]: `DELETE`?

File: server/src/cors.rs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+impl Fairing for CORS {
+    fn info(&self) -> Info {
+        Info {
+            name: "Add CORS headers to responses",
+            kind: Kind::Response,
+        }
+    }
+
+    async fn on_response<'r>(&self, _request: &'r Request<'_>, response: &mut Response<'r>) {
+        response.set_header(Header::new("Access-Control-Allow-Origin", "*"));
```
Comment (leeminkyu16) [inline server/src/cors.rs:17]: Classic

**Review by leeminkyu16 — APPROVED** [review]
Let's Gamble, Try Merging!

File: server/src/cors.rs (hunk 1 above)
Comment (andr-dev) [reply server/src/cors.rs:20]: True, will add in next PR.

**Pull request merged**